packages/chain-adapters/src/bitcoin/utxoSelect.ts

```diff
@@ -0,0 +1,72 @@
+import * as unchained from '@shapeshiftoss/unchained-client'
+import coinSelect from 'coinselect'
+import split from 'coinselect/split'
+
+import { bitcoin } from '../types'
+
+export type MappedUtxos = Omit<unchained.bitcoin.Utxo, 'value'> & { value: number }
+
+const handleStandardTx = ({
+  utxos,
+  value,
+  to,
+  satoshiPerByte,
+  sendMax
+}: {
+  utxos: unchained.bitcoin.Utxo[]
+  to: string
+  value: string
+  satoshiPerByte: string
+  opReturnData?: string
+  sendMax: boolean
+}) => {
+  const mappedUtxos: MappedUtxos[] = utxos.map((x) => ({ ...x, value: Number(x.value) }))
+  if (sendMax) {
+    return split(mappedUtxos, [{ address: to }], Number(satoshiPerByte))
+  } else {
+    return coinSelect<MappedUtxos, bitcoin.Recipient>(
+      mappedUtxos,
+      [{ value: Number(value), address: to }],
+      Number(satoshiPerByte)
+    )
+  }
+}
+
+// TODO this will be implemented in another PR to keep concerns separated and make review easier
+const handleOpReturnTx = ({
```

Comment:
May be easy enough to just push on the opReturn output if not undefined and just have a single function unless there are more differences that arise

Reply:
ya true that may be the case but for now this keeps it separate and its east enough to merge back into a single function if we find thats better